index.html

```diff
@@ -1 +1,147 @@
+						but switched to English A-levels before going to UCL again in London!
+						At UCL i studied Bio-chemical Engineering which taught me how to make pills from plasmids.
+						I needed a job (any job) and found myself being interviewed in the city and getting
+						a job as a junior accountant for an investment bank where I worked until the big move.
+				</p>
+			</div>
+			<div class="articleBox">
+				<img src="Img/bathview-KarimOwn.jpg" alt="View of Bath">
+				<p>My wife is from Bath and we would come back to visit family and friends.
+						We always talked about brining up our children out of London and couldn’t think of
+						a better place. So in Aug 2017 we picked up and moved 100miles down the M4 to junction 18.
+						For me this was a great opportunity for me to switch career to a new industry as
+						a full stack developer.</p>
+			</div>
+			<div class="articleBox">
+				<img src="Img/CyclingTrafSq.jpg" alt="Group of cyclists">
+				<p>This is me dressed as a MAMIL at the start of my 2 day cycle ride from London to Amsterdam.
+					I really enjoyed cycling 135 miles with a group of friends to raise money for charity.
+					I also support a financial educaiton charity as a Trustee.</p>
+			</div>
+        </section>
+	</div>
+	<div>
+		<section class="breatherContainer breatherText">
+			<h1>ABOUT ME</h1>
+			<p>Not quite my life story, nor all that I have done. This
+                focuses on my education and career jouney I have been on.</p>
+		</section>
+	</div>
+	<div class="backgroundImage portfolioImage" id="portfolioLocation">
+        <section class="container">
+            <div class="articleBox">
+                <img src="Img/challengeLogo.png" alt="Mayden Logo">
+                <p>Challenge was to copy the Mayden Logo using only CSS! None of this right click business!
+				This is harder than it sounds. Playing around with CSS shapes, border radius,
+				colo(u)rs and alignment. Trick is to use the inspector and arrows!</p>
+            </div>
+            <div class="articleBox">
+                <img src="Img/challengePilot.png" alt="Pilot Website">
+                <p>Copy a template website, functionality and
+                media queries for resizing depending on the
+                screensize.</p>
+            </div>
+            <div class="articleBox">
+                <img src="Img/challengeForm.png" alt="Input form">
+                <p>First Pair Coding work with James. This was a simple Form to build,
+				but difficult to navigate without diving onto the keyboard and driving!</p>
+            </div>
+        </section>
+	</div>
+	<div>
+		<section class="breatherContainer breatherText">
+			<h1>SOME STUFF I HAVE BUILT</h1>
+			<p>Started at Mayden Academy in Feb 2018. During my first 2 weeks
+				we were set a challenges which are shown above.
+				As the challenges get better (and the old ones get embarrassing), this
+				portfolio section will change!
+			</p>
```

Comment:
P tag placed differently from other places in the rest of the code.

Reply:
updated in 2 places